service/templates/layout.html

```diff
@@ -41,7 +41,7 @@
     <div class="phase-banner">
       <p>
         <strong class="phase-tag">BETA</strong>
-        <span>This is a new service - your <a href="mailto:digital-register-feedback@digital.landregistry.gov.uk">feedback</a> will help us to improve it.</span>
+        <span>This service is in private beta. Please give us your <a rel="external" href="mailto:digital-register-feedback@digital.landregistry.gov.uk">Feedback</a>.</span>
```

Comment:
I think it should still be - This is a new service - your feedback will help us improve it.
I'll try to find out where this change/decision has come from